examples/vis.py

```diff
@@ -8,13 +8,14 @@ def __init__(self):
         super().__init__(name='visualization', daemon=False)
         self.queue = deque(maxlen=10)
         self._lock = threading.Lock()
+        self.running = True
 
     def __call__(self, *args):
         with self._lock:
             self.queue.append(args)
     
     def run(self):
-        while True:
```

Comment:
code conflicts with the reverted PR https://github.com/trueagi-io/minecraft-demo/pull/43, please take a look

Reply:
I'd say that PR #43 conflicts with this code, but ok :)

Reply:
It was pr https://github.com/trueagi-io/minecraft-demo/pull/39, i wasn't aware that Oleg removed it.